tools/pkg-config/csfml-all.pc.in

```diff
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
+libdir=${exec_prefix}/@SFML_RELATIVE_INSTALL_LIBDIR@
```

Comment:
`SFML_RELATIVE_INSTALL_LIBDIR` doesn't exist in CSFML. Go look at that PR you linked to see where I defined that new variable and do something similar with CSFML.

Reply:
Yeah just saw that my bad :(

Reply:
What do you think of this?
It's heavily inspired by what was done on the SFML side